src/database/models/trip.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable valid-jsdoc */
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-unused-vars */
+/* eslint-disable strict */
+
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class Trips extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+     static associate(models) {
+      Trips.belongsTo(models.Accomodation,{foreignKey:'accomodationId'})
+      models.Accomodation.hasMany(models.Trips,{foreignKey:'accomodationId'});
+      models.Users.hasMany(models.Trips,{
+        foreignKey: {
+          name: 'userId'
+        }
+      });
+    }
+  }
+
+  Trips.init(
+    {
+      id: {
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+        allowNull: false,
+        primaryKey: true
+      },
+      userId:{
+        type: DataTypes.UUID,
+        allowNull: false,
+      },
+      accomodationId:{
+        type:DataTypes.INTEGER,
+        allowNull: false,
+      },
+      departure: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      destination: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      date_of_travel: {
```

Comment:
use Camel case notation to have a uniform in your table fields' the same way you did for accomodationId